ui/imports/shared/popups/UserAgreementPopup.qml

```diff
@@ -0,0 +1,121 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+StatusModal {
+    id: root
+    width: 640
+    headerSettings.title: qsTr("Welcome to Status Desktop Beta")
+    hasCloseButton: false
+
+    closePolicy: Popup.NoAutoClose
+
+    component Paragraph: StatusBaseText {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        font.pixelSize: 15
+        lineHeightMode: Text.FixedHeight
+        lineHeight: 22
+        visible: !!text
+        wrapMode: Text.Wrap
+    }
+
+    component AgreementSection: ColumnLayout {
+        property alias title: titleItem.text
+        property alias body: bodyItem.text
+        spacing: 12
+        Paragraph {
+            id: titleItem
+            font.weight: Font.DemiBold
+        }
+
+        Paragraph {
+            id: bodyItem
+        }
+    }
+
+    StatusScrollView {
+        id: scrollView
+        anchors.fill: parent
+        contentWidth: availableWidth
+
+        ColumnLayout {
+            id: layout
+            width: scrollView.availableWidth
+            spacing: 22
+            StatusRoundedImage {
+                id: statusRoundedImage
+                objectName: "headerImage"
+                Layout.preferredWidth: 64
+                Layout.preferredHeight: 64
+                Layout.alignment: Qt.AlignHCenter
+                image.source: Style.png("status-logo")
```

Comment:
adding `image.mipmap: true` here improves quality a bit bc the original image is much bigger than 64x64